src/plugins/intel_gna/src/gna_executable_network.hpp

```diff
@@ -130,6 +132,8 @@ class GNAExecutableNetwork : public InferenceEngine::IExecutableNetworkInternal
     InferenceEngine::Parameter GetMetric(const std::string& name) const override {
         if (ov::supported_properties == name) {
             return Config::GetSupportedProperties(true);
+        } else if (ov::model_name == name) {
+            return networkName;
         } else {
             return plg->GetMetric(name, {});
```

Comment:
`model_name` can be returned from 
`plg->GetMetric(name, {})`
as other metrics to make it more consistent

Reply:
Good idea - done